docs/user/manifests/kubeapps-local-dev-dex-values.yaml

```diff
@@ -119,3 +119,10 @@ volumes:
 volumeMounts:
   - mountPath: /etc/dex/tls/https/server
     name: https-tls
+resources:
+  limits:
+    cpu: 500m
+    memory: 256Mi
+  requests:
+    cpu: 50m
+    memory: 64Mi
```

Comment:
At least with Puppeteer, tests were failing many times while doing login with OIDC. Navigation would be lost at Dex side sometimes, so I decided to give it some resources. There were neither `limits` nor `requests` before.
Not sure if the change really had an effect, as many things were tried together (time constraints).
I'm totally fine with trying lower limits.

Reply:
Fine with those limits, just think it'd be good to communicate to our future selves that these numbers aren't determined empirically. Maybe just something like:

```
# TODO(castelblanque): Added more resources because navigation would be lost at Dex sometimes. Not yet determined
# this had an effect nor whether the values are accurate.
```

to make it easier for someone else to make a decision when working here.